src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphAuthDAOSpec.scala

```diff
@@ -250,21 +250,21 @@ class GraphAuthDAOSpec extends FlatSpec with Matchers with OrientDbTestFixture {
 
   it should "save Workspace Access Groups as map properties on a Workspace" in withDefaultTestDatabase { dataSource =>
     dataSource.inTransaction() { txn =>
-      val levels = authDAO.createWorkspaceAccessGroups(testData.workspace.toWorkspaceName, testUserInfo, txn)
-      val workspace = testData.workspace.copy(accessLevels = levels)
+      val levels = authDAO.createWorkspaceAccessGroups(testData.workspace2.toWorkspaceName, testUserInfo, txn)
+      val workspace = testData.workspace2.copy(accessLevels = levels)
       workspaceDAO.save(workspace, txn)
     }
 
     // separate transaction so we aren't checking un-saved vertices
     dataSource.inTransaction() { txn =>
-      withWorkspaceContext(testData.workspace, txn, bSkipLockCheck = true) { wc =>
+      withWorkspaceContext(testData.workspace2, txn, bSkipLockCheck = true) { wc =>
```

Comment:
Why were these workspace -> workspace2 changes made?  If it's to test on a Workspace with no existing access level groups, it should be named to make that clear.